### DISCUSSION TITLE
Besides Vercel, are there any other platforms supporting incremental static regeneration?

### DISCUSSION THREAD

**Discussion by pkellner** [vercel/next.js#19536] [Help]
I have about 2000 static pages and just a few added each day. Are there any other platforms besides vercel that supports that? I looked a Netlify but that was not clear on there site.  Any way to support this in a docker container?

**Comment by pkellner** [comment]
anyone home?

**Comment by timneutkens** [comment]
`next start` supports all Next.js features.
